### ISSUE TITLE
move instance's current rescue properties and methods to the rescuer 

### ISSUE THREAD

**Issue by runxinw** [GoogleCloudPlatform/gce-rescue#3]
move instance's current rescue properties and methods to the rescuer 

pls see commit: 802d9f713a2602e22a9bc4a4df4035a9741ae90a

**Comment by runxinw** [comment]
@dataclass
class Rescuer:
  """Initialize Rescuer instance."""
  rescuee: Instance 
  # _rescue_source_disk: str = ''
  # _rescue_mode_status: Dict[str, Union[str, int]] = field(
  #   default_factory=lambda: ({})
  # )

  # @property
  # def rescue_disk(self) -> str:
  #   return f'linux-rescue-disk-{self.ts}'


  # @property
  # def rescue_source_disk(self) -> str:
  #   return self._rescue_source_disk


  # @rescue_source_disk.setter
  # def rescue_source_disk(self, v: str) -> None:
  #   self._rescue_source_disk = v

**Comment by runxinw** [comment]
```
@dataclass
class Rescuer:
  """Initialize Rescuer instance."""
  rescuee: Instance 
  # _rescue_source_disk: str = ''
  # _rescue_mode_status: Dict[str, Union[str, int]] = field(
  #   default_factory=lambda: ({})
  # )

  # @property
  # def rescue_disk(self) -> str:
  #   return f'linux-rescue-disk-{self.ts}'


  # @property
  # def rescue_source_disk(self) -> str:
  #   return self._rescue_source_disk


  # @rescue_source_disk.setter
  # def rescue_source_disk(self, v: str) -> None:
  #   self._rescue_source_disk = v
```

**Comment by halleysouza** [comment]
fixed in commit e957c8f53136de0fa163eedb2b938e5b63d19f6d
